tests/wpt/meta/trusted-types/trusted-types-reporting-clipping-of-sample.html.ini

```diff
@@ -1,37 +1,37 @@
 [trusted-types-reporting-clipping-of-sample.html]
-  expected: TIMEOUT
+  expected: CRASH
```

Comment:
We run into https://github.com/w3c/webappsec-csp/issues/704 and https://github.com/w3c/trusted-types/issues/577 Seems like the spec is unresolved, so I think it is okay to crash for now.

Reply:
Let's file a followup issue for this so we track it!